### DISCUSSION TITLE
Code obfuscation after/before freeze code with cx-Freeze

### DISCUSSION THREAD

**Discussion by bw7715** [marcelotduarte/cx_Freeze#1361] [Q&A]
What ways of obfuscating code do you recommend after freezed your project with the cx-Freeze module?

Does cx-Freeze works with PyArmor, Cython, etc. ?

Is better to use some tools to obfuscate before or after freeze code with cx-Freeze to prevent reverse engineering?

<https://github.com/marcelotduarte/cx_Freeze/issues/595>
<https://github.com/marcelotduarte/cx_Freeze/issues/935>

**Comment by marcelotduarte** [comment]
pyamor referenced in #1268  but no more info

I use cython in some projects but it increase the build time and this method adds some work (the packages are not detected - I use the packages and includes options to specify them), and a coleteral effect: speed up ;-)
